File: csharp-project/Grill.cs
//this is a comment. it's not going to be interpreted by C# and is used to annotate important parts of the code
//classes are like blueprints. think of a factory - they need to know information on the parts needed to create a product
namespace seneca_project
{
    public class Grill
    {
        //property of temperature
        public int temperature
        {
            get;
            set;
        }
        public bool power
        {
            get;
            set;
        }
        public Grill(int temp = 0)
        { //constructor
            temperature = temp;
        }
        //used to increment and decrement the temperature
        public void increaseTemp(int temp)
        {
            temperature += temp;
        }
        public void decreaseTemp(int temp)
        {
            temperature -= temp;
        }
        public void turnOn()
        {
            power = true;
        }
        public void turnOff()
        {
            power = false;
        }
    }
}

File: src/main/java/com/senecafoundation/Grill.java
package com.senecafoundation;

public class Grill {
    public int temperature;
    public boolean power;

    public Grill() {
        // temperature = temp;
        // power = false;
    }

    static int MAX_POWER = 1000;

    static int MAX_TEMP = 500;

    public void increaseTemp(int temp) {
        temperature += temp;
    }
    public void decreaseTemp(int temp) {
        temperature -= temp;
    }
    public void turnOn() {
        power = true;
    }
    public void turnOff() {
        power = false;
    }
}